CooperUnion / vue-vuex-socket.io-opinionated-integration - README.md

yure-r opened the pull request:
README.md expanded to include serverside documentation.
You should totally merge it.

yure-r committed:
Update README.md

changed url to my website

yure-r committed:
Update README.md

added server-side documentation on two-way socket communication

yure-r committed:
Update README.md

slightly reformatted

yure-r committed:
Update README.md

commented out console logs

everyplace commented on README.md:
```diff
@@ -46,4 +46,23 @@ export default createStore({
+  
+  socket.on('vue_sendMessage', (data)=>{
+    console.log('vue_sendMessage received', data)
+    io.emit('user_message', data)
```
Can you change the io.emit to use socket_userMessage as well?

everyplace replied:
Or maybe it should be socket_systemMessage and we’ll update the vuex example too.

yure-r replied:
@everyplace  Updated!

yure-r committed:
Update README.md

changed io.emit('user_message', data) to 
 io.emit('socket_systemMessage', data) according to feedback
